src/main/java/uk/gov/hmcts/cmc/claimstore/events/operations/UploadOperationService.java

```diff
@@ -26,12 +26,15 @@ public UploadOperationService(DocumentUploadHandler documentUploadHandler,
     }
 
     public Claim uploadDocument(Claim claim, String authorisation, PDF document, CaseEvent caseEvent) {
+        Claim updatedClaim = uploadDocument(claim, authorisation, document);
+        return eventsStatusService.updateClaimOperationCompletion(authorisation, updatedClaim, caseEvent);
+    }
+
+    private Claim uploadDocument(Claim claim, String authorisation, PDF document) {
```

Comment:
do we need the private method?

Reply:
Inline Method refactoring is not supported when return statement interrupts the execution flow